src/main/java/dao/AirportDao.java

```diff
@@ -0,0 +1,20 @@
+package dao;
+
+import entities.Airport;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Set;
+
+/**
+ * DAO interface for airport entity
+ * @author klysov
+ */
+public interface AirportDao {
+    Airport extractAirportFromResultSet(ResultSet rs) throws SQLException;
```

Comment:
this method is not about DAO it's about its' realisation... I think it's better to make it privet end exclude from interface